packages/tradershub/src/components/Timeline/Timeline.classnames.ts

```diff
@@ -3,33 +3,33 @@ import { cva } from 'class-variance-authority';
 const stepperBaseStyles = 'z-10 box-border flex h-800 w-800 items-center rounded-pill outline outline-2';
 
 export const stepperVariants = cva(stepperBaseStyles, {
-    variants: {
-        isActive: {
-            true: 'transition-all delay-700 duration-700 ease-out  outline-solid-coral-700',
-            false: 'outline-solid-grey-5',
-        },
-        isFilled: {
-            true: '',
-        },
-    },
     compoundVariants: [
         {
+            class: 'bg-solid-grey-5',
             isActive: false,
             isFilled: true,
-            class: 'bg-solid-grey-5',
         },
         {
+            class: 'bg-solid-coral-700',
             isActive: true,
             isFilled: true,
-            class: 'bg-solid-coral-700',
         },
     ],
+    variants: {
+        isActive: {
+            false: 'outline-solid-grey-5',
+            true: 'transition-all delay-700 duration-700 ease-out  outline-solid-coral-700',
```

Comment:
I think the `transition-all delay-700 duration-700 ease-out` can be inside of the stepperBaseStyles.

Reply:
This is transition class it makes more sense if you applied it for both of the outline rather than just scoping it to one